src/it/java/teammates/it/sqllogic/core/NotificationsLogicIT.java

```diff
@@ -0,0 +1,70 @@
+package teammates.it.sqllogic.core;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.NotificationStyle;
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.it.test.BaseTestCaseWithSqlDatabaseAccess;
+import teammates.sqllogic.core.NotificationsLogic;
+import teammates.storage.sqlapi.NotificationsDb;
+import teammates.storage.sqlentity.Notification;
+
+/**
+ * SUT: {@link NotificationsLogic}.
+ */
+public class NotificationsLogicIT extends BaseTestCaseWithSqlDatabaseAccess {
+
+    private NotificationsLogic notificationsLogic = NotificationsLogic.inst();
+    private NotificationsDb notificationsDb = NotificationsDb.inst();
```

Comment:
Might be slightly cleaner to use getNotifications in logic instead of db, then we can remove this declaration.